ui/src/components/WMWorkerPoolsTable/index.jsx

```diff
@@ -100,24 +172,27 @@ export default class WorkerManagerWorkerPoolsTable extends Component {
         </TableCell>
 
         <TableCell>
-          <TableCellListItem
-            button
-            component={Link}
-            to={`${path}/providers/${workerPool.providerId}`}>
-            <ListItemText
-              disableTypography
-              primary={<Typography>{workerPool.providerId}</Typography>}
-            />
-            <LinkIcon size={iconSize} />
-          </TableCellListItem>
+          <Typography>
+            {workerPool.providerId !== NULL_PROVIDER && workerPool.providerId}
+          </Typography>
+        </TableCell>
+
+        <TableCell>
+          <IconButton
+            className={classNames(classes.button, classes.deleteButton)}
+            name={`${workerPool.workerPoolId}`}
+            onClick={this.handleDeleteClick}
+            disabled={actionLoading || workerPool.providerId === NULL_PROVIDER}>
+            <DeleteIcon size={iconSize} />
+          </IconButton>
```

Comment:
If we don't want the last column to be empty when a worker pool is scheduled for deletion, we could put the yellow label in that column instead of having it beside the worker pool ID. Your choice. I think both are okay.

Reply:
> I think a button being destructive doesn't necessarily need to be red

I personally like the red button. Hence its red color ;) I am not changing that.

> the construction icon is not universally known as "workers"

My opinion is that the meaning of the icon is pretty transparent. Clashing opinions is a dead end. Why don't we deploy this and have somebody to actually use it and get their feedfback? I.e. why don't we conduct an experiment and gather some actual data and actual user feedback? Not making these changes.

Feel free to refactor after this is merged.

Reply:
I thought not displaying any buttons for deleted workerpools is a good idea, so I'm not rendering Workers and Delete buttons for those. I also disable the editing link for them (viewing will be possible with the drawer thingy

Reply:
The background colors stay because I like them - besides, if you scroll this PR a bit up, you will see that the first iteration of the Edit button didn't have the background color, and Dustin missed the button. I added the color for visibility.

Reply:
I think the first iteration of the edit button was positioned in the same column as the worker pool ID which made it hard to distinguish from the worker pool ID link. It will be more visible to the eye now that it's in its own column and only displayed for rows that allow it.

re: the red background, I try to provide an explanation in https://github.com/taskcluster/taskcluster/pull/922 under the visual hierarchy section. Consistency is important to create a sense of control and to eliminate confusion. If any of the design choices don't make sense, I am more than happy to discuss. Upon agreement we can create an issue to change that aspect of the design choice on all views which will ensure consistency is maintained.

Reply:
> I think the first iteration of the edit button was positioned in the same column as the worker pool ID which made it hard to distinguish from the worker pool ID link

No, it was a separate column

Reply:
> No, it was a separate column

I was on PTO when there seemed to have been an issue with the first iteration of the edit button. I understood it was in the same column after reading https://github.com/taskcluster/taskcluster/pull/865#issuecomment-501459391:

> @djmitche I changed the Edit button styling - now it's blue, and I also put it into a separate column in a table to add spacing.

Reply:
> No, it was a separate column

I was on PTO when there seemed to have been an issue with the first iteration of the edit button. I got this from https://github.com/taskcluster/taskcluster/pull/865#issuecomment-501459391:

> @djmitche I changed the Edit button styling - now it's blue, and I also put it into a separate column in a table to add spacing.